README.md

```diff
@@ -24,10 +24,39 @@ A sample implementation of this can be found in the file `sample-main.go`
 and `pkg/sample-cmd` directory.  You'll want to have the equivalent files
 in your project.
 
-### A note on Dependencies
+### Building your own Custom Metric Server 
 
-You'll need to `glide install` dependencies before you can use this
-project.
+See [getting-started.md](docs/getting-started.md) for a walk through on creating your own custom metric server api.
```

Comment:
nit on wording: "creating your own implementation of the custom metrics API" or "creating you own custom metrics server".